### DISCUSSION TITLE
ruby installation problems in WSL on windows 11.  compiling iseq.c gcc: fatal error: Killed signal terminated program cc1

### DISCUSSION THREAD

**Discussion by PaulPatan** [rbenv/ruby-build#2367] [Build failures]
I tried installing Ruby inside WSL using both rbenv and asdf but both times I got `compiling iseq.c
gcc: fatal error: Killed signal terminated program cc1
compilation terminated.
make: *** [Makefile:448: enumerator.o] Error 1
make: *** Waiting for unfinished jobs....
external command failed with status 2`

**Comment by mislav** [comment]
Not sure what the cause is nor what the solution would be, but I seem to remember another discussion about compiler issues under WSL: https://github.com/rbenv/ruby-build/discussions/2298#discussioncomment-7603586
